source/scenario/mainMenu.vwf.yaml

```diff
@@ -17,7 +17,7 @@ extends: ../scenario/scenario.vwf
 properties:
   scenarioName: mainMenuScenario
 
-  nextScenarioPath: "introScreenScenario"
+  nextScenarioPath: "scenario1a"
```

Comment:
I changed it when I was getting a strange behavior upon changing scene.newGame (that is - without clearing the cache it would still enter introScreenScenario and I thought this was the issue). Probably inconsequential but I'm actually not sure if we can remove the mainMenu yaml